assets/training/finetune_acft_hf_nlp/src/register_model/register_model.py

```diff
@@ -245,14 +262,21 @@ def register_model(args: Namespace):
     # convert to safe tensors
     if args.convert_to_safetensors:
         convert_lora_weights_to_safetensors(args.model_path)
+        # update adapter_config.json with `peft_version`
+        update_peft_adapter_config(args.model_path)
```

Comment:
why are we conditioning update of adapter config with _convert_to_safetensors_?

Reply:
will keep it out